File: archive-old-practice/DAY 14/interface.cpp
#include<iostream>
using namespace std;
class connection{
    public:
    virtual void connect()=0;
};
class security{
    public:
    virtual void encrypt()=0;
};
class NetworkNode:public connection,public security{
    public:
    void connect(){
        cout<<"Node connected to network.";
    }
    void encrypt(){
        cout<<"Traffic encrypted via AES-256.";
    }
};
int main(){
    NetworkNode N;
    N.connect();
    N.encrypt();
    return 0;
}

File: DAY 14/interface.py
from abc import ABC, abstractmethod
class connection(ABC):
    @abstractmethod
    def connect(self):
        pass
class security(ABC):
    @abstractmethod
    def encrypt(self):
        pass
class NetworkNode(connection,security):
    def connect(self):
        print("Node connected to network.")
    def encrypt(self):
        print("Traffic encrypted via AES-256.")
N=NetworkNode()
N.connect()
N.encrypt()